CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ If we missed a note on some change or you have a questions on migrating from old
 feel free to ask us and community.
 
 ## 0.2.18 (UNRELEASED)
+1. fixed the getMany() result being droped randomly bug when using the buffer as primary key. ([#4220](https://github.com/typeorm/typeorm/issues/4220))
```

Comment:
This line should be in `Bug fixes` section.

Reply:
Sorry for that, i might misunderstand the contrib guide, which is saying I need to place this line inside the section unreleased, So i only searched for unreleased literally and placed this line below it.. LOL